app/services/search/query_builders/feed_content.rb

```diff
@@ -156,28 +162,36 @@ def range_keys
         end
       end
 
+      # Builds the `simple_query_string` query
+      # This query uses a simple syntax to parse and split the provided query
+      # string into terms based on special operators. The query then analyzes
+      # each term independently before returning matching documents.
+      # https://www.elastic.co/guide/en/elasticsearch/reference/7.10/query-dsl-simple-query-string-query.html
```

Comment:
Postgres FTS has ` websearch_to_tsquery` which behaves similarly. We'd need to check how to integrate this with the gem. 

https://www.postgresql.org/docs/11/functions-textsearch.html